src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,24 @@
-export const TodoInfo = () => {};
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo
+}
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
```

Comment:
it's a good practice to use object destructing
```suggestion
export const TodoInfo: React.FC<Props> = ({ todo }) => {
  const {...} = todo;
  
  return (
```